### DISCUSSION TITLE
Please Investigate if Uno Extension causes the IDE to freeze

### DISCUSSION THREAD

**Discussion by nextcodelab** [unoplatform/uno#8498] [General]
[https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022)

**Comment by jeromelaban** [comment]
Thanks for the report. It's unfortunately too vague for us to take action. Did you get a report telling you so?

**Reply by nextcodelab** [reply]
I repair latest preview visual studio 3 times.. to investigate this I tried to install uno only... Visual studio 2022 preview relaunching multiple times not responding..
![Screenshot 2022-04-05 084815](https://user-images.githubusercontent.com/73583588/161656721-d51d4af8-6c39-4d72-8ee6-8ee16e57f09a.png)

**Reply by nextcodelab** [reply]
All templates is loading...
![Screenshot 2022-04-05 085128](https://user-images.githubusercontent.com/73583588/161656853-d7a3d94e-c15d-4ecd-8329-d897afea8c17.png)

**Reply by nextcodelab** [reply]
> Thanks for the report. It's unfortunately too vague for us to take action. Did you get a report telling you so?

Here is my system  system my info.. I will post here the final results after the second test.
![Screenshot 2022-04-05 092503](https://user-images.githubusercontent.com/73583588/161661542-8cf9c8b3-c72e-4d95-a4f5-f2af75a01f3a.png)

Next test is.
1. Repair Visual studio then Install other extension and load project..

**Reply by nextcodelab** [reply]
> Thanks for the report. It's unfortunately too vague for us to take action. Did you get a report telling you so?

Final report.. after repairing VS preview and the remaining third party extensions still there...

The project successfully loaded.

![Screenshot 2022-04-05 100226](https://user-images.githubusercontent.com/73583588/161666335-a2f20c50-0a4e-43de-8176-88b38c7b3fce.png)


![Screenshot 2022-04-05 100942](https://user-images.githubusercontent.com/73583588/161667402-2f2e4d4a-e96a-45e1-8767-cb7f0f4cf562.png)


![Screenshot 2022-04-05 101655](https://user-images.githubusercontent.com/73583588/161666361-fe699e7e-71f0-4675-b9f2-9651e3bd526a.png)

The app debugging is responsive now without uno extension.
![Screenshot 2022-04-05 102505](https://user-images.githubusercontent.com/73583588/161666820-42e0995a-5245-4262-9fe7-2ac5b60c9676.png)

I NEED TO USE UNO FOR COMPLETE CUSTOMIZABLE CONTROLS FROM WINUI/UWP, Convert my project for specific design need.

**Reply by nextcodelab** [reply]
> Thanks for the report. It's unfortunately too vague for us to take action. Did you get a report telling you so?

Updates...
I RUN UNO APP THROUGH SAMPLE,,, 


![Screenshot 2022-04-05 170001](https://user-images.githubusercontent.com/73583588/161718818-04b432bf-9cd1-47d6-b7d4-e3d3abe4a4b0.png)

Navigation pane on android maybe should encrease opening time...

Thanks uno framework samples.

**Comment by nextcodelab** [comment]
maybe visual studio bugs because, after unsintalling the uno extensions I still have this...
```csharp
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.2.0-pre.2.1
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

C:\Program Files\Microsoft Visual Studio\2022\Preview>vsixinstaller /uninstall:UnoSolutionTemplate.VSIX.47605a66-fc91-4695-a86b-3478d4ec2788

C:\Program Files\Microsoft Visual Studio\2022\Preview>
```


![Screenshot 2022-04-05 090205](https://user-images.githubusercontent.com/73583588/161657745-594784c8-3628-4e6a-aae7-816d04635ae2.png)



I will try to restart the system..

**Reply by nextcodelab** [reply]
After uninstalling the uno extension and restarting the system the essues still there , my project is not loading...

I will try to repair the VS and install other single extension.
![Screenshot 2022-04-05 091440](https://user-images.githubusercontent.com/73583588/161658731-f60044ab-8f92-4a3e-82a1-27c34f789eb5.png)
